app/components/web-components/vl-modal-verify.js

```diff
@@ -1,54 +1,58 @@
-// TODO: octane-refactor
-/* eslint-disable ember/no-get */
-// eslint-disable-next-line ember/no-classic-components
-import Component from '@ember/component';
-import { inject } from '@ember/service';
-import { computed } from '@ember/object';
-
-// TODO: octane-refactor
-// eslint-disable-next-line ember/no-classic-classes, ember/require-tagless-components
-export default Component.extend({
-  intl: inject(),
-  message: null,
-  title: null,
-  showActions: true,
-  buttonType: 'danger',
-  showVerify: true,
-  buttonText: 'delete',
-
-  verifyButtonText: computed('intl', 'buttonText', function() {
-    return this.intl.t(this.get('buttonText'));
-  }),
-
-  // TODO: octane-refactor
-  // eslint-disable-next-line ember/no-component-lifecycle-hooks
-  didInsertElement() {
-    this._super(...arguments);
-    this.get('element').querySelector('[role="dialog"]')
-      .focus();
-  },
-
-  showDestructiveIcon: computed('buttonType', function() {
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+/**
+ * @param title {string}
+ * @param message {string}
+ * @param showActions {Boolean}
+ * @param showVerify {Boolean}
+ * @param buttonType {string}
+ * @param buttonText {string}
+ * @param isLoading {string}
+ * @param onVerify {Function}
+ * @param onCancel {Function}
+ */
+export default class WebComponentsVlModalVerify extends Component {
+  @service intl;
+
+  @tracked element;
+
+  get showActions() {
+    return this.args.showActions ?? true;
+  }
+
+  get showVerify() {
+    return this.args.showVerify ?? true;
+  }
+
+  get verifyButtonText() {
+    return this.intl.t(this.args.buttonText ?? 'delete');
+  }
+
+  get showDestructiveIcon() {
     return this.buttonType !== 'warning';
-  }),
+  }
 
-  isAlertButton: computed.equal('buttonType', 'danger'),
+  get buttonType() {
+    return this.args.buttonType ?? 'danger';
+  }
 
+  get isAlertButton() {
+    return this.buttonType === 'danger';
+  }
```

Comment:
I had to look twice at this one since `buttonType` is both from args and a getter.
Perhaps place the getter with `args` first and rename the getter slightly?